rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -597,4 +721,9 @@ private FederationStorageProvider createFederationStorageProvider(Repository rep
         StorageAccessor bridgeStorageAccessor = new BridgeStorageAccessorImpl(repository);
         return new FederationStorageProviderImpl(bridgeStorageAccessor);
     }
+
+    private FederationStorageProvider createFederationStorageProvider(StorageAccessor storageAccessor) {
+        return new FederationStorageProviderImpl(storageAccessor);
```

Comment:
Does this make sense? Doesn't hurt either but between calling this method or directly calling the constructor I don't see much difference

Reply:
Updated.